pom.xml

```diff
@@ -26,7 +26,7 @@
         <packaging.type>war</packaging.type>
         
         <reload4j.version>1.2.18.4</reload4j.version>
-        <flyway.version>9.21.2</flyway.version>
+        <flyway.version>9.22.1</flyway.version>
```

Comment:
Is the Flyway version change necessary? Should it or could it be a separate PR?

Reply:
in general I only bumped FlyWay to support newer versions of Postgres. if you like I can test without this change?

Reply:
Or you could update the title of this PR to also say "bump Flyway". No strong preference.

Reply:
Thanks for adding "bump flyway" to the title.